src/registrar.js

```diff
@@ -69,22 +71,20 @@ class Registrar {
   /**
    * Add a new connected peer to the record
    * TODO: this should live in the ConnectionManager
-   * @param {PeerInfo} peerInfo
+   * @param {PeerId} peerId
    * @param {Connection} conn
    * @returns {void}
    */
-  onConnect (peerInfo, conn) {
-    // TODO: This is not a `peer-info` instance anymore, but an object with the data.
-    // This can be modified to `peer-id` though, once `peer-info` is deprecated.
-    // if (!PeerInfo.isPeerInfo(peerInfo)) {
-    //   throw errcode(new Error('peerInfo must be an instance of peer-info'), ERR_INVALID_PARAMETERS)
-    // }
+  onConnect (peerId, conn) {
```

Comment:
`Connection` contains the PeerId `connection.remotePeer`, the params here can be consolidated.

Reply:
#611 is on the rescue for this 😛